packages/shared/src/test/helpers.ts

```diff
@@ -47,13 +47,9 @@ export function setupDatabaseTestSuite() {
 }
 
 export function setScryOutput<T>(output: T) {
-  (scry as MockedFunction<() => Promise<T>>).mockImplementationOnce(
-    async () => output
-  );
+  vi.mocked(scry).mockImplementationOnce(async () => output);
 }
 
 export function setScryOutputs<T>(outputs: T[]) {
-  (scry as MockedFunction<() => Promise<T>>).mockImplementation(
-    async () => outputs.shift()!
-  );
+  vi.mocked(scry).mockImplementation(async () => outputs.shift()!);
```

Comment:
[`mocked`](https://vitest.dev/api/vi.html#vi-mocked) is a no-op but gives us the type we want (and preserves the signature of the mocked function)